src/modules/career/online-profiles.rst

```diff
@@ -14,211 +14,68 @@ things to include, as well as things to leave out of each profile.
 Technical Resumes
 -----------------
 
-
-A resume is a **crucial** part of your job search. We’re not going to
-ramble on explaining what a resume is - it’s a part of the hiring
+A resume is a *crucial* part of your job search. We're not going to
+ramble on explaining what a resume is; it's a part of the hiring
 process for most industries. What we will focus on instead is how to
 create a technical resume that will highlight the skills you bring to a
 technical position.
 
 Technical Resume Format
 ^^^^^^^^^^^^^^^^^^^^^^^
 
-Let’s take a look at a resume we created at LaunchCode to show the
-format of an ideal technical resume.
-
-When it comes to the format of your resume, you need to quickly
-summarize your skills, give evidence of those skills, outline your
-learning journey so far, and then include everything else you would
-typically include with a resume.
-
-The clearly outlined sections of a technical resume:
-
-#. Personal Information
-#. Summary of Technical Skills
-#. Technical Experience
-#. Technical Education
-#. Other Experience
-#. Other Education
-
-Personal Information
-^^^^^^^^^^^^^^^^^^^^
-
-You must include your first and last name, a phone number where you can
-be reached, and a professional email address*.
-
-.. note::
-
-    A professional email address is some variation of your name.
-    a.lovelace@gmail.com gives a much more professional feeling than
-    fluffytoes32@yahoo.com. If you don’t have an email account for
-    professional communication, you should create one. We personally
-    recommend gmail.
-
-You should also include a link to your GitHub profile, a link to your
-LinkedIn, a physical address, and a link to your personal website (if
-you have one).
-
-Summary of Technical Skills
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
+For the Apprenticeship Program, we require you to complete a resume using `our template guide <https://docs.google.com/document/d/1Xr0O_ZnNizlWlUsxxCUZHL483NZWV7mmTCbWUhNVm74/view>`_. We require resumes of this format in order to enter our Apprenticeship program. Companies appreciate the shared template and it makes it easier for you to focus on the content of your resume rather than the style
```

Comment:
I'd change one or both of the mentions of 'apprenticeship program' to have the same capitalization. my preference is both lowercase for these kinds of names but you may have a more formal rule